### PR TITLE
feat(nimbus): Multi conclusion recommendation field

### LINKED ISSUE #10671: Enable UI to use multi recommendation field
┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-4503)

### PR DESCRIPTION
**Pull request by yashikakhurana**
Because

- We want to use a new conclusion recommendation field that allows user to select multiple recommendations

This commit

- Add the ability to select multiple (checkbox) for conclusion recommendations
- Removes field from the summary page to display (radio button)
-  Add a new option for `None` as in the old radio button if somebody selects `None` and saves it, it isn't saving it.
- Rename the conclusion recommendation config `conclusion_recommendations`  option to `conclusion_recommendations_choices` to avoid name conflict with the new `conclusion_recommendations` multi-select filed

Note: In the subsequent PR, will migrate the data- https://github.com/mozilla/experimenter/pull/10672 and remove the field altogether from frontend and backend

Old UI

Screenshot 2024-05-28 at 9 11 08 AM

New UI
Screenshot 2024-05-28 at 9 14 35 AM
Multiple badge
Screenshot 2024-05-29 at 10 00 27 AM

Fixes #10671

### CODE REVIEW

**Review by jaredlockhart — COMMENTED** [review]
I think we have to do this the other way around.  Ie migrate data into the new data structure, migrate the existing ui on to the new field, and then lastly move the ui to the new field.

If we do it in this order we could risk confusing users or losing data?

**Comment by yashikakhurana** [comment]
> er way around. Ie migrate data into the new data structure, migrate the existing ui on to the

sure once I get approval on both the PRs, we will merge migration one first, and then this one

**Review by jrbenny35 — APPROVED** [review]

**Comment by yashikakhurana** [comment]
Thank you @jrbenny35 for helping to fix the integration tests 🎉

**Pull request merged**
